com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java

```diff
@@ -148,20 +148,12 @@ public void initialize(IDebugAdapterContext context, Map<String, Object> props)
             ASTEvaluationEngine engine = new ASTEvaluationEngine(project, debugTarget);
             boolean newExpression = false;
             if (breakpoint != null) {
-                if (StringUtils.isNotBlank(breakpoint.getLogMessage())) {
```

Comment:
why this condition is deleted: StringUtils.isNotBlank(breakpoint.getLogMessage())?

Reply:
this is not needed any more. Previously the compiled expression for the logpoint and conditional breakpoint are saved separately, so when evaluating a breakpoint, need check whether it's a logpoint or conditional breakpoint. In the new implementation, they are saved in a map, and fetched from the map directly.